### DISCUSSION TITLE
Contributing to loxilb

### DISCUSSION THREAD

**Discussion by ragoragino** [loxilb-io/loxilb#71] [General]
Hey there, I would like to check in with you about the possibility to contribute to the project. I would love to contribute, though I have the impression that some of the issues are quite high-level and need extensive scoping. Would be eager to pick any such tasks up, though I think it would be great to discuss in detail possible approaches.

1) Is that the right way to contribute, i.e. working through the existing issues? Or do you have smaller pieces of work to pick up in the beginning?
2) If so, are there any means I can potentially chat with you either sync or async (I have tried joining your Slack channel, but it doesn't seem to be public?)

**Comment by ghost** [comment]
Hi,

It is nice to hear from you. And it would be really great if you could contribute to this project in any form.  Sorry about the glitch in joining the slack channel.  We are working out things to make it smooth for everyone. Would you kindly join as a member https://www.loxilb.io/members ?  We will automatically add you to slack channel afterwards !!
